### ISSUE TITLE
Deactivated user - remove ability to Add colleague and Send Message

### ISSUE THREAD

**Issue by KevinChagnon** [gctools-outilsgc/gcconnex#1674]
As a Deactivated member, I need to have the Add Colleague and Send Message option disabled on my profile so that I do not receive any type of notification or be added as a colleague by other site members.

However some options are still available to site members such as adding as a colleague and sending a message; We mention to site members that they would no longer receive notifications when they deactivate their account but there is no mention about site members being able to send them messages through the platform.

I believe this is an oversight. we shouldn't be able to send a site message to a deactivated account

## For the development team
- [ ] Issue user story documented
- [ ] UX input received
- [ ] Design completed
- [ ] Design validated by business team / UX
- [ ] Code review completed by peer
- [ ] Issue closing comment references any duplicate or connected issues or pull requests
- [ ] Issue closed

**Comment by KevinChagnon** [comment]
https://github.com/Phanoix/gcconnex-help/issues/50

**Comment by klecuyer** [comment]
Possibly linked to https://github.com/gctools-outilsgc/gcconnex/issues/1652

**Comment by sabrinatbs** [comment]
Testing this issue, I deactivated my account. Becky helped me by sending messages, adding me as a colleagues and adding me to a group. I received an email notification for all of these. I reactivated my account and none of my "Personal Notifications" are checked. This issue still seems to be occurring 
![notification emails](https://user-images.githubusercontent.com/35535700/35860513-353d3112-0b3c-11e8-978d-178f27d46e1d.PNG)
